addon/components/paper-dialog.js

```diff
@@ -68,18 +68,22 @@ export default Component.extend({
   didInsertElement() {
     this._super(...arguments);
     if (this.get('escapeToClose')) {
-      $(this.get('destinationId')).on(`keydown.${this.elementId}`, (e) => {
+
+      this._destinationEle = document.querySelector(this.destinationId);
```

Comment:
I think we should use `this.get('destinationId')` since `destinationId` is a computed property.